recipes-qt/qt5/qtbase_git.bb

```diff
@@ -237,6 +237,10 @@ do_install_append() {
     echo "isEmpty(QMAKE_STRIP): QMAKE_STRIP = ${TARGET_PREFIX}strip" >> $conf
 
     generate_target_qt_config_file ${D}${OE_QMAKE_PATH_BINS}/qt.conf
+
+    # Fix up absolute paths in scripts
+    sed -i -e '1s,#!.*,#!${USRBINPATH}/env python,' \
+        "${D}${OE_QMAKE_PATH_QT_ARCHDATA}/mkspecs/features/uikit/devices.py"
```

Comment:
Please use same expression as in use in master:

```
+    # Fix up absolute paths in scripts                                                                                                                                                                            
+    sed -i -e '1s,#!/usr/bin/python,#! ${USRBINPATH}/env python,' \                                                                                                                                               
+        ${D}${OE_QMAKE_PATH_QT_ARCHDATA}/mkspecs/features/uikit/devices.py 
```

This is better as it does not replace _any_ shebang but a specific one. Do the same in nativesdk recipe.

Reply:
Take a look at `sed -i -e '1s,#!.*,#!${USRBINPATH}/env python,'` as it seems to have not been updated.